packages/schematics/src/command-line/affected.ts

```diff
@@ -277,6 +277,14 @@ async function runCommand(
   errorMessage: string
 ) {
   if (parsedArgs.parallel) {
+    // Make sure the `package.json` has the `ng: "ng"` command needed by `npm-run-all`
+    const packageJson = fs.readFileSync('./package.json').toString('utf-8');
+    if (!JSON.parse(packageJson).scripts.ng) {
```

Comment:
check that the scripts property is there. You might have a package.json file without having any scripts.

Even though it isn't easy to invoke the affected command without npm scripts, it's possible